packages/web/src/common.css

```diff
@@ -18,6 +18,11 @@
   color-scheme: light dark;
 }
 
+html.controlled-theme {
+  /* Override default color-scheme. Prevent conflicts with system color scheme since we have manual controls to change the page theme */
```

Comment:
Previously we were using the default color scheme of the system until user forced a theme manually. We have code for that of the form 
```
  const prefersDarkScheme = window.matchMedia('(prefers-color-scheme: dark)');
  const defaultTheme = prefersDarkScheme.matches ? 'dark-mode' : 'light-mode';
```

Is this code no longer needed? Do we not align with the OS settings anymore?

Reply:
UI Mode only reflected the system color scheme on first launch. On first launch, that media query is evaluated, the `defaultTheme` written to `localStorage`, and the media query ceased to be involved in the calculation from that point on.

Specifying `color-scheme: light dark` would be correct for a system that is completely responsive to the system color scheme, but prior to this PR it was not being properly overridden. We were applying our own theme, but `color-scheme` was still doing whatever it wanted from the system, thus resulting in lesser customized things like scrollbars and checkboxes being colored with the system theme regardless of our theme selection.

Reply:
> I Mode only reflected the system color scheme on first launch. On first launch, that media query is evaluated, the defaultTheme written to localStorage, and the media query ceased to be involved in the calculation from that point on.

Could you share a code pointer?

Reply:
It is the snipped you stated.

Fetch OS theme and use it as default. Combine with any existing theme setting. Apply class.

```
// applyTheme()
const prefersDarkScheme = window.matchMedia('(prefers-color-scheme: dark)');
const defaultTheme = prefersDarkScheme.matches ? 'dark-mode' : 'light-mode';

const currentTheme = settings.getString('theme', defaultTheme);
if (currentTheme === 'dark-mode')
  document.body.classList.add('dark-mode');
```

At this point the displayed theme on the page reflects the result of this calculation. The global `color-scheme: light dark` no longer applies (except to cause problems).